tests/contrib/test_rest_framework.py

```diff
@@ -97,3 +97,28 @@ def test_to_representation(self, image_upload_file, settings):
                 }
             },
         }
+
+    @pytest.mark.django_db
+    def test_to_representation__with_aspect_ratios(self, image_upload_file, settings):
+        settings.PICTURES["USE_PLACEHOLDERS"] = False
+
+        profile = models.Profile.objects.create(picture=image_upload_file)
+        serializer = ProfileSerializer(profile)
+        serializer.fields["picture"].ratio = "16/9"
+        serializer.fields["picture"].breakpoints = {"m": 4, "l": 3}
+
+        assert serializer.data["picture"] == {
+            "sources": {
+                "WEBP": {
```

Comment:
I wonder if this shouldn't be the MIME-type. I'd need it anyway in the HTML. That way, you wouldn't even need to know what is being served.

Reply:
OK, how would you limit the result set? Wasn't that kinda the problem?

Reply:
Media only works, if you provide some kind of grid info. It's also identical to all sources.

Reply:
But that's already a case - in your last commit, payload could be reduced by not using all aspect ratios. I don't know how we could approach `sources` as I am lacking understanding the need of so many variants...